test/api/user/is-github-organization-admin.test.js

```diff
@@ -74,6 +74,19 @@ test('GET `/user/is-github-organization-admin` 401 unauthorized | no github id',
   t.deepEqual(res.statusCode, 401)
 })
 
+test('GET `/user/is-github-organization-admin` 401 unauthorized | not github admin', async (t) => {
+  t.context.github.isUserAnOrgAdmin.resolves(false)
+  const res = await t.context.app.inject({
+    method: 'GET',
+    url: '/user/is-github-organization-admin',
+    query: { orgId: `${t.context.orgId1}` },
+    headers: {
+      cookie: `${USER_WEB_SESSION_COOKIE}=${t.context.session1}`
+    }
+  })
+  t.deepEqual(res.statusCode, 401)
+})
```

Comment:
hmm actually I think I would have guessed this route to return 200 { isAdmin: false } or something so that the caller doesn't have to catch the status code. wdyt ?

Reply:
actually i agree. i feel 401 implies no perms to hit endpoint, not that the endpoint logic executed and returned false